test/libgit2.jl

```diff
@@ -793,5 +793,4 @@ mktempdir() do dir
             end
         end
     =#
-    end
```

Comment:
this will probably be a syntax error and not bisect, should be squashed into whichever commit added it